tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-start-end.html

```diff
@@ -94,11 +94,11 @@
       el.selectionStart = 0;
       el.selectionEnd = 5;
       el.selectionStart = 8;
-      assert_equals(el.selectionStart, 8, `selectionStart on ${el.id}`);
-      assert_equals(el.selectionEnd, 8, `selectionEnd on ${el.id}`);
+      assert_equals(el.selectionStart, 5, `selectionStart on ${el.id}`);
+      assert_equals(el.selectionEnd, 5, `selectionEnd on ${el.id}`);
       el.remove();
     }
-  }, "Setting selectionStart to a value larger than selectionEnd should increase selectionEnd");
+  }, "Setting selectionStart to a value larger than selectionEnd should set selectionStart to selectionEnd");
```

Comment:
I presume you saw that text in [set selection range](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#set-the-selection-range) step 3? That unfortunately isn't the correct place to look; on the [`selectionStart`'s setter's algorithm](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#textFieldSelection:dom-textarea/input-selectionstart-2) step 3, we can clearly see that end is supposed to be set to the given value, BEFORE running set selection range.

Reply:
Right, thanks, I did miss that nuance. Fixed now.